### PR TITLE
Plans/Checkout: Selected Feature

### PR DESCRIPTION
**Pull request by rodrigoi**
Selected feature query string parameter is passed down the checkout chain to the thank you page.

### CODE REVIEW

**Comment by retrofox** [comment]
@rodrigoi need rebase

File: client/components/plans/plan-actions/index.jsx [hunk 1]
```diff
@@ -139,9 +143,14 @@ const PlanActions = React.createClass( {
 			return this.props.onSelectPlan( cartItem );
 		}
 
+		let checkoutPath = '/checkout/' + this.props.site.slug;
```
Comment (retrofox) [inline client/components/plans/plan-actions/index.jsx]: Just FYI you can use string templates here which is pretty good.

```es6
let checkoutPath = `/checkout/${ this.props.site.slug }`;
```

**Comment by retrofox** [comment]
we should consider how we are passing the current product. 
https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/plans/index.js#L33

**Comment by roundhill** [comment]
LGTM, but... conflicts now :(

**Pull request merged**